Makefile-docker

```diff
@@ -39,27 +40,26 @@ timeago/jquery.timeago.js \
 underscore/underscore.js \
 
 NODE_LIBS_JQUERY_UI := \
-jquery-ui/ui/version.js \
 jquery-ui/ui/data.js \
 jquery-ui/ui/disable-selection.js \
+jquery-ui/ui/focusable.js \
 jquery-ui/ui/ie.js \
+jquery-ui/ui/jquery-patch.js \
 jquery-ui/ui/keycode.js \
-jquery-ui/ui/escape-selector.js \
 jquery-ui/ui/labels.js \
-jquery-ui/ui/jquery-1-7.js \
 jquery-ui/ui/plugin.js \
+jquery-ui/ui/position.js \
 jquery-ui/ui/safe-active-element.js \
 jquery-ui/ui/safe-blur.js \
 jquery-ui/ui/scroll-parent.js \
-jquery-ui/ui/focusable.js \
 jquery-ui/ui/tabbable.js \
 jquery-ui/ui/unique-id.js \
-jquery-ui/ui/position.js \
+jquery-ui/ui/version.js \
 jquery-ui/ui/widget.js \
-jquery-ui/ui/widgets/menu.js \
-jquery-ui/ui/widgets/mouse.js \
 jquery-ui/ui/widgets/autocomplete.js \
 jquery-ui/ui/widgets/datepicker.js \
+jquery-ui/ui/widgets/menu.js \
+jquery-ui/ui/widgets/mouse.js \
```

Comment:
renamed jquery-1.7.js to jquery-patch.js; dropped escape-selector.js and then reordered the list alphabetically